core/src/main/java/com/xmartlabs/bigbang/core/controller/SessionController.java

```diff
@@ -19,15 +19,15 @@
  * Thus, the {@link SessionType} object is serialized using {@link Gson}.
  * The session is retrieved once from disk and then kept in memory for faster access.
  */
-public abstract class SessionController<S extends SessionType> extends Controller {
+public abstract class SessionController extends Controller {
```

Comment:
this should not change, you should add something like `public abstract class SessionController extends Controller implements "SomeInterface"`